corehq/apps/consumer_user/decorators.py

```diff
@@ -1,14 +1,15 @@
 from functools import wraps
 from django.contrib.auth.views import redirect_to_login
 from django.urls import reverse
+from corehq.apps.consumer_user.models import ConsumerUser
 
 
-def login_required(view_func):
+def consumer_user_login_required(view_func):
     @wraps(view_func)
     def _inner(request, *args, **kwargs):
         user = request.user
-        if not (user.is_authenticated and user.is_active):
-            url = reverse('consumer_user:patient_login')
+        if not (user.is_authenticated and user.is_active and ConsumerUser.objects.get_or_none(user=user)):
+            url = reverse('consumer_user:consumer_user_login')
             return redirect_to_login(request.get_full_path(), url)
 
         # User's login and domain have been validated - it's safe to call the view function
```

Comment:
I think you copy / pasted this from elsewhere, but it's confusing here. A `ConsumerUser` doesn't have a "domain", right?

Reply:
Fixed the comment